Patch-Source: https://github.com/davidhalter/jedi/commit/ab97e9f78443fdab784a61bccd0c34bed103a332
--
From ab97e9f78443fdab784a61bccd0c34bed103a332 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lum=C3=ADr=20=27Frenzy=27=20Balhar?=
 <frenzy.madness@gmail.com>
Date: Wed, 22 May 2024 23:13:19 +0200
Subject: [PATCH] Fix compatibility with pytest 8

Fixes #1995
---
 test/test_utils.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/test_utils.py b/test/test_utils.py
index f17fc2462..2416dc6bd 100644
--- a/test/test_utils.py
+++ b/test/test_utils.py
@@ -12,7 +12,7 @@ class TestSetupReadline(unittest.TestCase):
     class NameSpace(object):
         pass
 
-    def __init__(self, *args, **kwargs):
+    def setUp(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
         self.namespace = self.NameSpace()
